web/package.json

```diff
@@ -152,6 +152,7 @@
         "lodash-es": "4.17.21",
         "lru-cache": "8.0.4",
         "mini-svg-data-uri": "1.4.4",
+        "moment-timezone": "^0.5.43",
```

Comment:
Please remove the ^ and do yarn again 🙏🏻 If we don't, everyone's local can have different versions and it gets hard to troubleshoot local issues